File: tests/fixtures/r/test_smells.r
test_addition <- function() {
  result <- add(1, 2)
  stopifnot(result == 3)
}

test_subtraction <- function() {
  result <- subtract(5, 3)
  stopifnot(result == 2)
}

test_multiplication <- function() {
  result <- multiply(4, 5)
  stopifnot(result == 20)
}

File: tests/fixtures/objc/test_smells.m
#import <Foundation/Foundation.h>

void test_addition(void) {
    NSInteger result = 0;
    for (NSInteger i = 0; i < 10; i++) {
        result += i * i;
        if (result > 50) {
            result = 50;
        }
    }
    NSLog(@"result: %ld", (long)result);
}

void test_subtraction(void) {
    NSInteger result = 0;
    for (NSInteger i = 0; i < 10; i++) {
        result += i * i;
        if (result > 50) {
            result = 50;
        }
    }
    NSLog(@"result: %ld", (long)result);
}
